### PR TITLE
Gc articles bugfixes

### PR DESCRIPTION
**Pull request by andrewleith**
# Summary | Résumé

Bug fixes for the test/hack session.

## QA
- [x] relative links used on some pages (why-notify, features)
- [x] Links in formatting guide should not leave site
- [x] "Why notify" menu item not active when on page
- [x] Image on `/message-delivery-status` pointing to staging
- [x] "Your account" menu going underneath blue hero banner on home page
- [x] Mobile language switch not working
- [x] Incorrect links on `/security` page
- [x] "More menu" (seen in tablet view) going underneath blue hero banner on home page

### CODE REVIEW

**Comment by andrewleith** [comment]
Better question: where is that menu? :) How do I get there?

**Comment by jimleroyer** [comment]
The images from the `/message-delivery-status` page now points to production. Is that what we want? We didn't want these to target on staging env from review/staging environments targeting staging?

File: app/assets/javascripts/moreMenu.js [hunk 1]
```diff
@@ -62,6 +62,8 @@
       .children(":gt(" + fitCount + ")")
       .not("#more-menu");
 
+    collectedSet.find(" > div").addClass("z-50");
```
Comment (jimleroyer) [inline app/assets/javascripts/moreMenu.js:65]: hmm that could break easily

**Comment by jimleroyer** [comment]
The more menu fix is a bit messed up compared to production:

In review environment:

image

In production environment:

image

**Comment by andrewleith** [comment]
> The images from the `/message-delivery-status` page now points to production. Is that what we want? We didn't want these to target on staging env from review/staging environments targeting staging?

Both notify staging and notify PROD point to GCA PROD, which I think is correct.  

What we should probably do, however, is create a notify-staging site in GCA prod, and point our staging to it so we can make changes there that don't affect our prod content.

What we didn't want is to have any our environments pointing to a staging version of GCA, since it would not be reliable.

**Comment by andrewleith** [comment]
> The more menu fix is a bit messed up compared to production:
> 
> In review environment:
> 
> image
> 
> In production environment:
> 
> image

Looks like i targeted the wrong element in my fix, just fixed it again, please re-check!

**Comment by amazingphilippe** [comment]
Menus looks good on my end:
Capture d’écran, le 2022-05-26 à 14 00 30
Capture d’écran, le 2022-05-26 à 14 00 44

**Comment by andrewleith** [comment]
@jimleroyer good on your end?

**Comment by jimleroyer** [comment]
> Both notify staging and notify PROD point to GCA PROD, which I think is correct.

> What we should probably do, however, is create a notify-staging site in GCA prod, and point our staging to it so we can make changes there that don't affect our prod content.

> What we didn't want is to have any our environments pointing to a staging version of GCA, since it would not be reliable.

Historically, we had staging environments to target staging assets (and still have for most content not served by GCArticles). 

Practically, targeting production assets in staging environment will not be handy for testing nor consistency (we'll have assets from both staging and prod environments mixed together in the staging environment).

I understand that's because we are now serving GCArticles that is in production, hence that production targets production assets. That makes sense... I wonder if there is no way out of this though to keep testability and consistency. That will be harder to test new assets that way.

I can approve this though as this is a current design limitation and outside the scope of this PR.

**Review by jimleroyer — APPROVED** [review]
:shipit:

**Pull request merged**